### PR TITLE
glusterdocs: Move Resolving-Peer-Rejected to Troubleshooting

### PR DESCRIPTION
**Pull request by nbalacha**
Consolidate all topics dealing with resolving issues in the Troubleshooting section.

Signed-off-by: N Balachandran <nbalacha@redhat.com>

### CODE REVIEW

File: Troubleshooting/resolving-peer-rejected.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
-Peer Rejected is a state a peer can be in. If you see a peer in this
-state when you run 'gluster peer status' that means volume configuration
+##Fixing "Peer Rejected"
```
Comment (prashanthpai) [inline Troubleshooting/resolving-peer-rejected.md]: A general note: If there isn't a space between `#` and start of the heading text, [GitHub doesn't render it as a heading](https://github.com/github/markup/issues/1013). You will see this broken across all documentation currently. As this is a new code, please add the space.

We should think of a way to automate bulk fixing this in all files.

File: Troubleshooting/resolving-peer-rejected.md [hunk 2]
```diff
@@ -11,9 +17,12 @@ On the rejected peer:
     UUID file)
 3.  Start glusterd
 4.  Probe one of the good peers
+
+                gluster peer probe <server>
+
 5.  Restart glusterd, check 'gluster peer status'
 6.  You may need to restart glusterd another time or two, keep checking
```
Comment (prashanthpai) [inline Troubleshooting/resolving-peer-rejected.md]: Just a side note: From a user's perspective, statements in line 24 to 28 does not instil confidence in Gluster :(

Comment (nbalacha) [reply Troubleshooting/resolving-peer-rejected.md]: That was the existing documentation - I just moved it to a different location. I plan to work with the glusterd folks to get it done properly.

File: Troubleshooting/resolving-peer-rejected.md (hunk 1 above)
Comment (nbalacha) [reply Troubleshooting/resolving-peer-rejected.md]: Fixed for this instance.

**Pull request merged**